web/crux-ui/src/hooks/use-websocket.ts

```diff
@@ -1,12 +1,19 @@
+import { defaultWsErrorHandler } from '@app/errors'
 import { isServerSide } from '@app/utils'
 import { WebSocketClient, WebSocketClientOptions, WebSocketEndpoint } from '@app/websockets/client'
+import useTranslation from 'next-translate/useTranslation'
 import { useEffect, useRef, useState } from 'react'
 
 let client: WebSocketClient
 
 export const useWebSocket = (route: string, options?: WebSocketClientOptions): WebSocketEndpoint => {
+  const { t } = useTranslation()
```

Comment:
as we discussed, we will have to make a provider for the WebSocketClient, so the TFunction only gets obtained once.